app-text/dblatex/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<longdescription lang="en">
+	</longdescription>
+</pkgmetadata>
+
```

Comment:
Plus `<herd>proxy-maintainers</herd>`, unless somoene else volunteers to maintain it.